### PR TITLE
website update & copy edits

### PR DESCRIPTION
**Pull request by abbey-titcomb**
Draft PR for #258 

Signed-off-by: abbey titcomb <abbeytitcomb@gmail.com>

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -146,21 +143,22 @@ <h4>Own your infrastructure</h4>
...
-                 </svg> Build your own value flows</p>
+                 </svg> 
+  Build your own value flows with "DevFi" lego blocks like donations, tips, and automated payouts</p>
```
Comment (brandonhaslegs) [inline docs/index.html:161]: what the hell does this mean 😂

**Review by brandonhaslegs — CHANGES_REQUESTED** [review]
Here are some suggestions and comments. Also I think we need to reevaluate the icons in each section. Some are repeated and some just don't match with the content they're next to.

File: docs/index.html [hunk 2]
```diff
@@ -55,9 +55,9 @@ <h1 class="headline druk">Free your code</h1>
           <source type="video/mp4" src="/img/extending-hands.mp4">
         </video>
 
-        <h1 class="druk">Say hello to your new workspace</h1>
+        <h1 class="druk">Code collaboration + Web3 = 🌱</h1>
```
[the remark this answers is not in the document]
Comment (brandonhaslegs) [reply docs/index.html:58]: We might as well say "GitHub for crypto" … this has the same energy to me.

[the remark this answers is not in the document]
Comment (lftherios) [reply docs/index.html:58]: We wanted something that talks more to our decentralized audience. That's why we went for that. I think it does the job of explaining the angle of Radicle.

File: docs/index.html [hunk 3]
```diff
@@ -55,9 +55,9 @@ <h1 class="headline druk">Free your code</h1>
...
         <div class="subtitle-container">
-          <h2>A decentralized app for code collaboration</h2>
+          <h2>For unstoppable, permissionless, and decentralized open-source development</h2>
```
[the remark this answers is not in the document]
Comment (lftherios) [reply docs/index.html:60]: hm I don't have a good substitute.

File: docs/index.html [hunk 4]
```diff
@@ -93,23 +90,23 @@ <h4>Collaborate peer-to-peer</h4>
 
           <div class="attribute work-offline">
             <div class="content">
-              <h4>Work securely offline</h4>
-              <p>No need to trust anyone to keep your work safe or online.</p>
+              <h4>Manage your codebases on-chain</h4>
+              <p>Decentralize your open source project with Ethereum-powered Radicle Orgs.</p>
```
[the remark this answers is not in the document]
Comment (lftherios) [reply docs/index.html:94]: yeap

File: docs/index.html [hunk 5]
```diff
@@ -93,23 +90,23 @@ <h4>Collaborate peer-to-peer</h4>
...
                    <path d="M27.5858 17.586C28.3669 16.805 29.6332 16.805 30.4142 17.586L32.9637 20.1355C33.2942 20.0471 33.6416 20 34 20C36.2091 20 38 21.7909 38 24C38 26.2091 36.2091 28 34 28C31.7909 28 30 26.2091 30 24C30 23.6417 30.0471 23.2945 30.1354 22.9641L27.5858 20.4144C26.8048 19.6334 26.8048 18.3671 27.5858 17.586Z"/>
                    <path d="M23.9999 19C25.1045 19 25.9999 19.8954 25.9999 21V30.5351C27.1956 31.2267 28 32.5194 28 34C28 36.2091 26.2091 38 24 38C21.7909 38 20 36.2091 20 34C20 32.5195 20.8044 31.2268 21.9999 30.5352V21C21.9999 19.8954 22.8954 19 23.9999 19Z"/>
                  </svg>
- Built on Git</p>
+ Share ownership of repositories with multi-sigs</p>
```
[the remark this answers is not in the document]
Comment (lftherios) [reply docs/index.html:104]: I think it's important to explain ownership of what.

File: docs/index.html [hunk 6]
```diff
@@ -93,23 +90,23 @@ <h4>Collaborate peer-to-peer</h4>
...
                    <path d="M24.5859 23.5858C25.3669 22.8047 26.6332 22.8047 27.4143 23.5858L39.4143 35.5858C40.1953 36.3668 40.1953 37.6332 39.4143 38.4142C38.6332 39.1953 37.3669 39.1953 36.5858 38.4142L35.0001 36.8284L31.4143 40.4142C30.6332 41.1953 29.3669 41.1953 28.5858 40.4142C27.8048 39.6332 27.8048 38.3668 28.5858 37.5858L32.1716 34L31.0001 32.8284L27.4143 36.4142C26.6332 37.1953 25.3669 37.1953 24.5858 36.4142C23.8048 35.6332 23.8048 34.3668 24.5858 33.5858L28.1716 30L24.5859 26.4142C23.8048 25.6332 23.8048 24.3668 24.5859 23.5858Z"/>
                  </svg>
- Backed by public-key cryptography</p>
+ Secure and anchor project state on-chain</p>
```
[the remark this answers is not in the document]
Comment (lftherios) [reply docs/index.html:109]: Works.

File: docs/index.html [hunk 7]
```diff
@@ -146,21 +143,22 @@ <h4>Own your infrastructure</h4>
           <div class="attribute create-community">
             <div class="content">
               <h4>Create with your <br/> community</h4>
```
[the remark this answers is not in the document]
Comment (lftherios) [reply docs/index.html:145]: I still prefer the "create with your community"

File: docs/index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (brandonhaslegs) [reply docs/index.html:58]: How about:

Your decentralized web3 workspace

image

File: docs/index.html [hunk 8]
```diff
@@ -146,21 +143,22 @@ <h4>Own your infrastructure</h4>
...
                  </svg>
- Collectively manage codebases</p>
+ Create and fund developer communities with NFT-based membership models</p>
```
[the remark this answers is not in the document]
Comment (lftherios) [reply docs/index.html:150]: Works

File: docs/index.html (hunk 1 above)
Comment (lftherios) [reply docs/index.html:161]: hehe fair. we can stick with the old version

**Review by brandonhaslegs — APPROVED** [review]

**Pull request merged**